jest.config.js

```diff
@@ -40,6 +41,7 @@ module.exports = {
   setupFilesAfterEnv: ['<rootDir>/test/jest/setup.js'],
   testMatch: [
```

Comment:
Nice we'll now be including:
- app/scripts/controllers/app-metadata.test.ts
- app/scripts/controllers/encryption-public-key.ts

Q: 
do we have plans to include:
- app/scripts/controllers/metametrics.test.js
- app/scripts/controllers/swaps.test.js
?

Reply:
Good question! But it'll soon be moot, I'm rewriting and vastly simplifying how this works in the very next PR (here: https://github.com/MetaMask/metamask-extension/pull/25013)